tensorflow/tsl/platform/cpu_info.h

```diff
@@ -134,6 +134,13 @@ enum CPUFeature {
   AMX_BF16 = 43,  // Bfloat16 tile matrix multiplication
 };
 
+enum Aarch64_CPU {
```

Comment:
Nit: Let's stick with the camel case enum naming.
```suggestion
enum Aarch64CPU {
```

Reply:
Fixed in https://github.com/tensorflow/tensorflow/pull/61110/files